.env.template

```diff
@@ -2,5 +2,10 @@ export POSTGRES_USER="forum_example_postgraphile"
 export POSTGRES_PASSWORD="xyz"
 export POSTGRES_DB="forum_example_postgraphile"
 export DATABASE_SCHEMAS="forum_example"
-export DATABASE_URL="postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@localhost/${POSTGRES_DB}"
+export DATABASE_URL="postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@localhost:5432/${POSTGRES_DB}"
```

Comment:
Out of interest, why was this added? It should be implicit, adding it feels like writing `http://google.com:80/` or `https://google.com:443/` but if it's necessary under some circumstances I'd love to know why so I can incorporate it into the docs.

Reply:
Nope, it's not necessary. I didn't notice that I added the port at some point during development. We can take it or leave it.

Reply:
Okidokey. I might clean it up at some point